src/main/blackbox/blackbox.c

```diff
@@ -1795,8 +1793,10 @@ STATIC_UNIT_TESTED bool blackboxShouldLogIFrame(void)
 #ifdef USE_GPS
 STATIC_UNIT_TESTED bool blackboxShouldLogGpsHomeFrame(void)
 {
-    if ((GPS_home[0] != gpsHistory.GPS_home[0] || GPS_home[1] != gpsHistory.GPS_home[1]
-        || (blackboxPFrameIndex == blackboxIInterval / 2 && blackboxIFrameIndex % 128 == 0)) && isFieldEnabled(FIELD_SELECT(GPS))) {
+    if ((GPS_home_llh.lat != gpsHistory.GPS_home.lat
+         || GPS_home_llh.lon != gpsHistory.GPS_home.lon
+         || (blackboxPFrameIndex == blackboxIInterval / 2 && blackboxIFrameIndex % 128 == 0))
```

Comment:
```suggestion
        || GPS_home_llh.lon != gpsHistory.GPS_home.lon
        || (blackboxPFrameIndex == blackboxIInterval / 2 && blackboxIFrameIndex % 128 == 0))
```

Reply:
The indentation is intentionally to second parenthesis - `||` part, different from `&&` alignment .. 
It's not perfect, but alternatives seemed even worse to me (add whitespace to first line (`f ((GPS_home_llh.lat` ) or put parentheses on separate lines)

Reply:
I tend to stick to tab size spacing for indentation. As seeing this style used on python.

Having coded a little in COBOL in the eighties indentation was mandatory :joy:

Reply:
Even Emacs agrees with me on this one ;-)
With 8-space indentation, this would be a bit easier. I prefer to see the logic, aligning to 4 spaces would hide that (it was easier in lisp).